cmd/ssh_usercert_gen/main.go

```diff
@@ -64,9 +65,9 @@ type AppConfigFile struct {
 }
 
 const (
-	AuthLevelNone     = -1
-	AuthLevelPassword = 0
-	AuthLevelU2F      = 1
+	AuthLevelNone     = 0
+	AuthLevelPassword = 1
+	AuthLevelU2F      = 2
```

Comment:
Do you think of these as true "levels" (i.e. bigger enum implies "better")? If so, more forethought seems necessary, as perhaps later something just as strong (not better, not weaker) as U2F may come along.

We can avoid this conundrum if instead these are called `AuthType*`. If needed, we can later add a strength mapping. For admin control, instead of specifying a "minimum strength level" instead they could provide a bitmask/list of supported auth types.